tests/e2e/infra.tf

```diff
@@ -0,0 +1,92 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+module "vpc" {
+  source = "terraform-aws-modules/vpc/aws"
+
+  name = "${var.enviroment_id}"
+
+  cidr = "10.0.0.0/16"
+
+  azs = [
+    "us-east-1a",
+    "us-east-1b",
+    "us-east-1c",
+  ]
+
+  private_subnets = [
```

Comment:
are these really required for current test? Just for my understanding, if we dont create private/public subnets will aws actuator fail to create instances for machine objects?